playbooks/operationalizing/cloudforms.adoc

```diff
@@ -114,6 +114,8 @@ CloudForms can be deployed into all types of environments. A full https://access
 
 The CloudForms appliance is delivered as a virtual image template that can be installed on several platforms including OpenStack, Red Hat Enterprise Virtualization and in the VMWare vSphere environment. Consult the link:https://access.redhat.com/documentation/en/red-hat-cloudforms/[installation documentation] for platform specific directions outside of the materials provided here.
 
+Once the machine is up and available, use ssh to login using the username/password combination of `root/smartvm`
+
 Configure CloudForms and database connectivity according to the documentation and to your specific environment. 
```

Comment:
Add here that they need to use the 'appliance_console' command to enter the configuration utility